builder/linode/builder_test.go

```diff
@@ -12,7 +12,7 @@ import (
 func testConfig() map[string]interface{} {
 	return map[string]interface{}{
 		"linode_token":  "bar",
-		"region":        "us-east",
+		"region":        "us-ord",
 		"instance_type": "g6-nanode-1",
 		"ssh_username":  "root",
 		"image":         "linode/alpine3.9",
```

Comment:
Some images may also need an upgrade, maybe let's use Debian 12 for all of them?